client/js/widgets/widget.js

```diff
@@ -1370,27 +1374,35 @@ export class Widget extends StateManaged {
       }
 
       if(a.func == 'ROTATE') {
-        setDefaults(a, { count: 1, angle: 90, mode: 'add', collection: 'DEFAULT' });
+        setDefaults(a, { count: 1, angle: 90, mode: 'add', target: 'DEFAULT' });
         let collection;
         const mode = a.mode == 'set' ? 'to' : 'by';
         if(a.holder !== undefined) {
-          if(this.isValidID(a.holder, problems)) {
-            await w(a.holder, async holder=>{
-              for(const c of holder.children().slice(0, a.count || 999999))
-                await c.rotate(a.angle, a.mode);
-            });
+          if(collection = getCollection(a.holder)) {
+            for(const w of collections[collection]){
+              if (w.childrenTarget().length){
+                for(const c of w.childrenTarget().slice(0, a.count || 999999))
+                  await c.rotate(a.angle, a.mode);
+              } else {
```

Comment:
I'm not sure if it should be an error if there is nothing to ROTATE in the specified holders, just like it's not an error if there is nothing to MOVE. It seems like this would generate errors for expected states sometimes.

Reply:
I added this because we get a similar error for collections. You are correct that it might generate a bunch of "empty holder" errors that have close to no value.

I'm open to remove it